content/chat/rooms/messages.textile

```diff
@@ -566,6 +567,89 @@ h3(#ordering-update-delete). Ordering Updates and Deletes
 
 Applying an action to a message produces a new version, which is uniquely identified by the @version@ property. When two message instances share the same @serial@ they represent the same chat message, but they can represent different versions. Lexicographically sorting the two message instances by the @version@ property gives the global order of the message versions: the message instance with a greater @version@ is newer, the message instance with a lower @version@ is older, and if their @version@ is equal then they are the same version.
 
-The @Message@ object also has convenience methods "@versionBefore@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#versionbefore, "@versionAfter@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#versionafter and "@versionEqual@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#versionEqual which provide the same comparison.
+The @Message@ object also has convenience methods "@isOlderVersionOf@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#isolderversionof, "@isNewerVersionOf@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#isnewerversionof and "@isSameVersionAs@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Message.html#issameversionas which provide the same comparison.
 
 Update and Delete events provide the full message payload, so may be used to replace the entire earlier version of the message.
+
+h3(#keep-messages-updated). Keep messages updated with @with@
```

Comment:
At the moment this is in the "ordering" section - feels like it should be part of message update?

Reply:
Moved to a h2.